hardhat.config.ts

```diff
@@ -111,11 +52,83 @@ const config: HardhatUserConfig = {
       },
     ],
   },
+  networks: {
+    hardhat: isFork(),
+    development: {
+      url: "http://127.0.0.1:8545/",
+      chainId: 31337,
+      live: false,
+    },
+    bsctestnet: {
+      url: "https://bsc-testnet.public.blastapi.io",
+      chainId: 97,
+      live: true,
+      gasPrice: 20000000000,
+      accounts: {
+        mnemonic: process.env.MNEMONIC || "",
+      },
+    },
+  },
+  gasReporter: {
+    enabled: process.env.REPORT_GAS !== undefined,
+    currency: "USD",
+  },
+  etherscan: {
+    apiKey: {
+      bsc: process.env.ETHERSCAN_API_KEY || "ETHERSCAN_API_KEY",
+      bsctestnet: process.env.ETHERSCAN_API_KEY || "ETHERSCAN_API_KEY",
+    },
+  },
+  paths: {
+    tests: "./test",
+  },
+  // Hardhat deploy
+  namedAccounts: {
+    deployer: 0,
+    acc1: 1,
+    acc2: 2,
+    proxyAdmin: 3,
+    acc3: 4,
+  },
   docgen: {
     outputDir: "./docs",
     pages: "files",
     templates: "./docgen-templates",
   },
+  external: {
+    contracts: [
+      {
+        artifacts: "node_modules/@venusprotocol/oracle/artifacts",
+      },
+      {
+        artifacts: "node_modules/@venusprotocol/venus-protocol/artifacts",
+      },
+    ],
+    deployments: {
+      bsctestnet: ["node_modules/@venusprotocol/oracle/deployments/bsctestnet"],
+    },
```

Comment:
I think we don't need this config

Reply:
Fixed !
Right, no need of this dependency in oracle, if requires in future we will add them back.